### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by brean** [allenh1/rplidar_ros#7]
ROS2 uses `colcon`, the `README.md` still referes to `catkin`

**Comment by allenh1** [comment]
> ROS2 uses colcon, the README.md still referes to catkin

Yes, that's true. I never updated the documentation, since I was hoping to get this sent upstream first.

I'd be more than happy to accept a PR updating the `README.md` if you have some time!

**Comment by brean** [comment]
Thanks, I'll create a PR on the weekend.

**Comment by brean** [comment]
Sorry, I was a bit to optimistic, I could not get it running on the weekend. I am now working on a setup to run ROS (and ROS2) in docker container for better testing and understand ros2, so feel free to update the documentation yourself, cause this might take some time 😉

**Comment by chuan9** [comment]
@allenh1 hi, allenh
I'm using dashing version for ros2, and it won't make success after run command "colcon build", will you help update your tutorial here?
the log is bellows:
```
 warning: invalid suffix on literal; C++11 requires a space between literal and string macro [-Wliteral-suffix]
```

**Comment by brean** [comment]
@chuan9 that is only a warning, it should still compile. Do you have a longer log and/or do you get another error-message? Also can you provide the exact steps you did or provide your updated Dockerfile?

**Comment by chuan9** [comment]
@brean Actually, the compilation is ok. I changed some paras and bingo, it works on S1

**Comment by allenh1** [comment]
> @brean Actually, the compilation is ok. I changed some paras and bingo, it works on S1

Good to hear! Let me know if you need any further assistance.
